doc/ledger3.texi

```diff
@@ -4686,6 +4689,10 @@ normal ledger file (specified via @option{--file @var{FILE} (-f)} or via
 the environment variable @env{LEDGER_FILE}) this entry will not be
 printed again.
 
+As you can see in output above, the account is @samp{Expenses:Unknown}
```

Comment:
Done. But from my point of view, I do not dislike in a manual to be taken by hand by the author to explain me things. I agree then for personal style. I do not understand "personal opinion" related to "As you can see in output above" that is a personal style to introduce fact. :-)